lib/LedgerSMB/Scripts/asset.pm

```diff
@@ -912,6 +912,39 @@ CSV formats.
 
 =cut
 
+sub _import_file {
```

Comment:
is this the best naming for this function?
It appears the function is specific to importing a CSV file, so perhaps the function name should reflect that